lib/exercism/homework.rb

```diff
@@ -19,11 +19,9 @@ def status(language)
     exercises = user.exercises.where(language: language)
 
     skipped = exercises.where.not(skipped_at: nil).pluck(:slug)
-    fetched = exercises.where.not(fetched_at: nil).pluck(:slug)
+    fetched = exercises.where('fetched_at IS NOT NUll AND iteration_count = ?', 0).pluck(:slug)
```

Comment:
If it is skipped, they may not have fetched it. I think we should also exclude anything where skipped_at has a value other than nil.